src/main/java/com/autotune/analyzer/experiment/ExperimentResultValidation.java

```diff
@@ -79,7 +87,7 @@ public void validate(List<ExperimentResultData> experimentResultDataList, Map<St
                         if (null != kruizeObject.getResultData())
                             isExist = kruizeObject.getResultData().contains(resultData);
                         if (isExist) {
-                            errorMsg = errorMsg.concat(String.format("Experiment name : %s already contains result for timestamp : %s", resultData.getExperiment_name(), resultData.getEndtimestamp()));
+                            errorMsg = errorMsg.concat(String.format("Experiment name : %s already contains result for timestamp : %s", resultData.getExperiment_name(), resultData.getEndtimestamp().toInstant()));
```

Comment:
Why is this being converted to `toInstant`? Will the user see the same date String as what they sent in?

Reply:
Yes. This is how the response is coming now:

```
{
    "message": "Experiment name : quarkus-resteasy-autotune-min-http-response-time-db already contains result for timestamp : 2022-01-26T06:40:48.511Z",
    "httpcode": 409,
    "documentationLink": "",
    "status": "ERROR"
}
```

Reply:
It was missing the `T` and `Z` in the timestamp response while converting to string.

`"message": "Experiment name : quarkus-resteasy-autotune-min-http-response-time-db already contains result for timestamp : 2022-01-26 12:10:48.511",`

Reply:
By using the `java.sql.Timestamp` class, we're able to get the Timezone correctly if there's no offset value i.e. if it's like this `2022-01-26T12:00:00.511Z` . But this fails for the cases when there's an offset like this added to it, `2022-01-26T12:00:00.511+03:30` . 

One of the solution I found and tested was if we switch to `java.time.OffsetDateTime` class from the  `Timestamp` class